WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -776,6 +807,9 @@ protected void onSaveInstanceState(Bundle outState) {
         outState.putBoolean(STATE_KEY_HTML_MODE_ON, mHtmlModeMenuStateOn);
         outState.putSerializable(WordPress.SITE, mSite);
         outState.putParcelable(STATE_KEY_REVISION, mRevision);
+        outState.putSerializable(STATE_KEY_EDITOR_SESSION_DATA, mPostEditorAnalyticsSession);
+        // don't call sessionData.end() in onDestroy() if this is an Android config change
+        mIsConfigChange = true;
```

Comment:
Nit: let's put the comment on the same line as `mIsConfigChange = true;` since it only refers to that line and not for the following two.

Reply:
👍 addressed in 37ae831bd8